tests/host_reference/gsmear_test_helpers.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
Do we need that as a separate header?

Reply:
fixed, enumerator is moved into a destination header